src/Entity/RPG.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use App\Repository\RPGRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: RPGRepository::class)]
+class RPG
```

Comment:
I'd just name this class "Game", for self-explanatory's sake. It's usually good to avoid acronyms
(to be fixed everywhere it's mentioned)